pkg/feeds/testing/lookup.go

```diff
@@ -17,10 +19,30 @@ import (
 	"github.com/ethersphere/bee/pkg/feeds"
 	"github.com/ethersphere/bee/pkg/storage"
 	"github.com/ethersphere/bee/pkg/storage/mock"
+	"github.com/ethersphere/bee/pkg/swarm"
 )
 
+type Timeout struct {
+	storage.Storer
+}
+
+var searchTimeout = 30 * time.Millisecond
+
+// Get overrides the mock storer and introduces latency
+func (t *Timeout) Get(ctx context.Context, mode storage.ModeGet, addr swarm.Address) (swarm.Chunk, error) {
+	ch, err := t.Storer.Get(ctx, mode, addr)
+	if err != nil {
+		if errors.Is(err, storage.ErrNotFound) {
+			time.Sleep(searchTimeout)
+		}
+		return ch, err
+	}
+	time.Sleep(time.Duration(rand.Intn(10)) * time.Millisecond)
```

Comment:
I would increase this significantly, perhaps to 100 or more milliseconds delay. Also, this assures that chunks which are always returned before the chunks which are not found. Theoretically OK, but does not give us any test coverage that can simulate jitter

Reply:
I think it is ok, i tested and it does what it is meant to  do. 
increasing the time 10 fold would just slow down tests